README.md

```diff
@@ -1,24 +1,144 @@
-# README
+## Rales Engine Project
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+### Description
 
-Things you may want to cover:
+Nick and Kyle's [Rales Engine](http://backend.turing.io/module3/projects/rails_engine) project for Turing. In this project we import business data and host an API with endpoints to return records, relationships between models, and business intelligence as JSON. Checkout the schema file `schema.png` in the public directory to see the models and their relationshps.
 
-* Ruby version
+Tech stack: Ruby, Rails, PostgreSQL, RSpec
 
-* System dependencies
+### Dependencies 
 
-* Configuration
+This project uses Ruby version 2.3.3 with a PostgreSQL database. The dataset is from the [sales engine](https://github.com/turingschool-examples/sales_engine/tree/master/data) repository.
```

Comment:
I'm not sure that we specify the ruby version anywhere, and 2.3.3 is awfully specific.  I just checked, and I'm using 2.3.0.  Maybe we suggest ruby 2.3+  ?  Thoughts?  Also, to include the ruby version, we could include it in the Gemfile like: `ruby '~> 2.3.0'` which would allow it to float from 2.3.0, up to 2.3.9.  Another way would be to create a `.ruby-version` file (note the leading 'dot', and it's a dash, not underscore).  The contents of the `.ruby-version` file would simply be: `2.3.3`  (that's it).